.changeset/sharp-ducks-attack.md

```diff
@@ -0,0 +1,5 @@
+---
+"@thirdweb-dev/react": major
```

Comment:
this is definitely not a major version, `patch` should be totally fine

```suggestion
"@thirdweb-dev/react": patch
```

Reply:
Yes I am new changeset😅😅